src/messages/MMDSCacheRejoin.h

```diff
@@ -24,6 +24,7 @@
 #include "mds/CInode.h"
 #include "mds/CDir.h"
 #include "mds/mdstypes.h"
+#include "mds/MDCache.h"
```

Comment:
message's member functions should be simple, it's bad to call MDCache's member functions

Reply:
thanks for the review, will handle it later.
i think it's not finished yet, i got  make check failure,  it seems one of the test case is not passed.